src/filters/madgwickahrs.cpp

```diff
@@ -42,7 +42,7 @@ namespace dynamicgraph
 
     /* --- DG FACTORY ---------------------------------------------------- */
     DYNAMICGRAPH_FACTORY_ENTITY_PLUGIN(MadgwickAHRS,
-				       "MadgwickAHRS");
+                                      "MadgwickAHRS");
 
```

Comment:
Please make sure that you are not introducing a tab here.
I apologize because it looks like my editor did not indicate properly that we have moved to tabs.
But this is wrong.
On good editors like sublime text, tabs are being interpreted in a different way unlike spaces.
This bad when helping our students.
Could you make sure that this are not tabs. After untabifying this file the place of MadgwickAHRS should be aligned.

Reply:
?!? I took great care in not introducing tabs. In fact, there where *a lot* of tabs in the original code, and I removed them. All of the whitespace changes you are seeing is me removing tabs. I really hate tabs, and I am pretty sure I removed all of them. But I'll double check

Reply:
I confirm there are no tabs. Also, I removed additional ones in the header